manifests/profile/hathitrust/ingest_jobs.pp

```diff
@@ -0,0 +1,178 @@
+# Copyright (c) 2018 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+
+# nebula::profile::hathitrust::ingest_jobs
+#
+# Manage the ingest systemd service for hathitrust
+#
+# @example
+#   include nebula::profile::hathitrust::ingest_jobs
+class nebula::profile::hathitrust::ingest_jobs(
+  String $config,
+  String $feed_home,
+  Array[String] $dcu_recipients,
+  String $crms_renewals_source,
+  String $crms_renewals_dest,
+  String $recipient,
+  String $rights_db_config,
+  String $rights_log = '/tmp/populate_rights.log'
+) {
+
+  $feed_perl = "perl -I ${feed_home}/lib ${feed_home}/bin"
+  $feed_log  = "${feed_home}/var/log"
+  $joined_dcu_recipients = $dcu_recipients.join(',')
+  $base_env = [
+    "MAILTO=${recipient}",
+    "FEED_HOME=${feed_home}"
+  ]
+
+
+  cron {
+    default:
+      environment => $base_env + [ "HTFEED_CONFIG=${config}" ],
+      user        => 'libadm';
+
+    ## HOURLY  JOBS
+
+    # Pickup rights from CRMS & aleph and populate the rights database
+    'CRMS rights pickup':
+      # TODO: install loadrights script
+      command     => "${feed_perl}/feed.hourly/populate_rights_data.pl --pickup \
```

Comment:
This concat construct is a little strange to me in that it's part bin and part path to the thing to run. I guess, since there is the `-I`, it might not work cleanly as shebangs in the scripts themselves. I wonder, though, if a base runner under bin that does any includes and takes a path of the "real script" as the first param simplifies things (here and for any manual commands).

Reply:
I don't disagree, but I also don't think that making the change is in scope for right now.

Reply:
Fair.